src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -134,6 +139,45 @@ public static IEnumerable<LowLevelClientMethod> BuildMethods(RestClientBuilder b
             }
         }
 
+        private LowLevelConvenienceMethod BuildConvenienceMethod(in LowLevelClientMethod clientMethod, RestClientBuilder builder)
+        {
+            List<Parameter> parameters = new List<Parameter>();
+            MethodSignature protocolSignature = clientMethod.Signature;
+            InputOperation operation = clientMethod.RequestMethod.Operation;
+            Parameter? bodyParameter = null;
+            foreach (var parameter in protocolSignature.Parameters)
+            {
+                if (parameter == KnownParameters.RequestContent || parameter == KnownParameters.RequestContentNullable)
+                {
+                    bodyParameter = builder.GetBodyParameter(operation, DefaultNamespace);
+                    parameters.Add(bodyParameter!);
+                }
+                else if (parameter == KnownParameters.RequestContext)
+                {
+                    parameters.Add(KnownParameters.CancellationTokenParameter);
```

Comment:
Do we know that `RequestContext` is always the last parameter?

Reply:
Currently, it is. https://github.com/Azure/autorest.csharp/blob/66570505a5724751905560bfc8357fbf1834c81e/src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs#L282